### PR TITLE
Add landing page starting point

### PR DESCRIPTION
**Pull request by keriwarr**
Run it by opening index.html in your browser

Lot's of things could be organized better - it was implemented fairly hastily :)

svgs should probably go in separate files...

### CODE REVIEW

File: landing_page/js/plugins.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Avoid `console` errors in browsers that lack a console.
```
Comment (kingscott) [inline landing_page/js/plugins.js]: Is this file necessary?

File: landing_page/index.html [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+<html class="no-js" lang="">
+
+<head>
+  <meta charset="utf-8">
+  <title>housingWR</title>
+  <!-- <meta name="description" content=""> -->
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!-- <link rel="apple-touch-icon" href="icon.png"> -->
+  <!-- Place favicon.ico in the root directory -->
```
Comment (kingscott) [inline landing_page/index.html]: Does @skchuhan have a favicon we can add to this?

File: landing_page/index.html [hunk 3]
```diff
@@ -0,0 +1,467 @@
...
+  <link rel="stylesheet" href="css/main.css">
+  <link
+    href="https://fonts.googleapis.com/css2?family=Arvo:wght@400;700&family=Montserrat:wght@700&family=Open+Sans&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <div class="column" style="max-width: 1120px; margin: 20px
+  auto 0 auto; position: relative;">
+    <svg width="1151" height="38" viewBox="0 0 1151 38" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (kingscott) [inline landing_page/index.html]: I think it would be helpful if the SVGs were in separate files, as previously mentioned.

This comments goes for the others in this file :)

File: landing_page/index.html [hunk 4]
```diff
@@ -0,0 +1,467 @@
...
+  </div>
+  <div class="row footer">
+    <div class="column" style="justify-content: center;">
+      <p class="aboutus">About Us</p>
+      <p style="font-size: 16px;">We are an intiative started from CivicTechWR. </p>
+    </div>
+    <div class="column" style="justify-content: center;">
+      <div class="row">
+        <!-- TODO: -->
+        <a href="#">Documentation</a>
```
Comment (kingscott) [inline landing_page/index.html]: I think we can remove this line.
```suggestion

```

File: landing_page/css/normalize.css [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+
+/**
+ * Correct the inability to style clickable types in iOS and Safari.
+ */
+
+button,
+[type="button"],
+[type="reset"],
+[type="submit"] {
+  -webkit-appearance: button;
```
Comment (kingscott) [inline landing_page/css/normalize.css:199]: I would add a rule here to be a bit more explicit:

```css
cursor: pointer;
```

File: landing_page/index.html [hunk 6]
```diff
@@ -0,0 +1,467 @@
...
+        <clipPath id="clip0">
+          <rect width="1151" height="38" fill="white" />
+        </clipPath>
+        <clipPath id="clip1">
+          <rect width="149" height="38" fill="white" />
+        </clipPath>
+      </defs>
+    </svg>
+
+    <div class="row" style="margin-top: 94px;">
```
Comment (kingscott) [inline landing_page/index.html]: I'm thinking it may be a good idea to clean up some of the sizing and margins in this file. If they are standardized a bit (ex. margins using increments of 4px, etc) I think that'll help with the mobile responsiveness.

**Review by kingscott — CHANGES_REQUESTED** [review]
Overall it's looking pretty good Keri! I left a few small comments that should be easy to fix. 

I think the only thing I'm looking to improve here is some of the mobile responsiveness. Here's a screenshot from Chrome (simulating an iPhone SE): 

image

If you have any other questions, don't hesitate to reach out.

**Comment by keriwarr** [comment]
> I think the only thing I'm looking to improve here is some of the mobile responsiveness. Here's a screenshot from Chrome (simulating an iPhone SE):


oh yea, I stopped working on the mobile responsiveness once you indicated interest in working on that yourself :)

File: landing_page/js/plugins.js (hunk 1 above)
Comment (keriwarr) [reply landing_page/js/plugins.js]: s'pose not

File: landing_page/index.html (hunk 2 above)
Comment (keriwarr) [reply landing_page/index.html]: I think it's pending

File: landing_page/index.html (hunk 4 above)
Comment (keriwarr) [reply landing_page/index.html]: I think Sukhi may have a URL to go here?

File: landing_page/css/normalize.css (hunk 5 above)
Comment (keriwarr) [reply landing_page/css/normalize.css:199]: I'm inclined to leave the vendor code as in. I may add such a rule in main.css

File: landing_page/index.html (hunk 6 above)
Comment (keriwarr) [reply landing_page/index.html]: Maybe I got a bit carried away with directly copying values from the figma 😆

**Comment by skchuhan** [comment]
favicon! 
[favicon_io.zip](https://github.com/CivicTechWR/improved_housing_portal/files/5352069/favicon_io.zip)

File: landing_page/index.html (hunk 2 above)
Comment (kingscott) [reply landing_page/index.html]: Looks like we can add this in now :)

File: landing_page/index.html (hunk 4 above)
Comment (kingscott) [reply landing_page/index.html]: I wasn't sure what documentation we'd have, but if that's the case - sure!

File: landing_page/css/normalize.css (hunk 5 above)
Comment (kingscott) [reply landing_page/css/normalize.css:199]: Sounds good!

**Comment by kingscott** [comment]
I did, didn't I? :D sorry about that. Whenever some of the changes get made just re-tag me and then I can start on the mobile responsiveness part. Thanks @keriwarr!

**Comment by keriwarr** [comment]
@kingscott I did a really basic pass on the mobile responsiveness. It's not gorgeous but it's functional. Please feel free to take it from here if you wish. I don't know what's up with the preview image you posted above though... at the time, what I saw on equivalent settings was this: 
image

**Comment by keriwarr** [comment]
### quick preview of mobile experience

ezgif com-video-to-gif
